lib/logging/include/SWO.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace Logger {
+
+// C-style printf - prints to SWO interface
+void SWO_printf(const char* format, ...);
+
+// SWO logger
+class SWO {
```

Comment:
i think itd be nice to have a utility folder and in there we can put this Logger. the class should be called Logger i think. Utility::Logger.